tests/tensor_search/test_add_documents.py

```diff
@@ -1371,3 +1371,83 @@ def test_params_validation_non_tensor_fields_only(self):
 
         assert params.tensor_fields is None
         assert params.non_tensor_fields == ["field1", "field2"]
+
+
+class TestAddDocumentsUtils(MarqoTestCase):
+    def test_create_chunk_metadata(self):
```

Comment:
It is worth having a test to ensure that the error message renders into a string properly (i.e. `abc.__name__` gets interpreted correctly

Reply:
What error message are you referring to?

Reply:
Basically the new validation error messages. For example:
- https://github.com/marqo-ai/marqo/blob/61ec1cc9829411ad6c1e35227925974ba10fe4bb/src/marqo/tensor_search/validation.py#L466
- https://github.com/marqo-ai/marqo/blob/61ec1cc9829411ad6c1e35227925974ba10fe4bb/src/marqo/tensor_search/validation.py#L601
- https://github.com/marqo-ai/marqo/blob/61ec1cc9829411ad6c1e35227925974ba10fe4bb/src/marqo/tensor_search/validation.py#L567

There are a few reason to test these:
1. Ensure that the formatting is correct (and the variables are being replaced with the correct string)
2. Ensure that formatting code doesn't have any errors. This is quite important. For example, if you are trying to generate an error message for a 4xx error, but you have some error in the generation code, this will return a confusing 500 error to the user
3. Some users will have logic based on the error message (against best practices). For example:  `if "index not found" in error.message`. For these users, changes to error messages are technically breaking changes. We may decide to make these breaking changes, but we at least need to detect a change to the error message. Our error messages are an integral part of our public API

Reply:
Understood, will add these tests!